src/messages/formatters.ts

```diff
@@ -230,7 +230,7 @@ export enum Faces {
 	/**
 	 * ¯\\_(ツ)_/¯
```

Comment:
Ehm... I'm late, but shouldn't this be updated as well?
```suggestion
	 * ¯\\_(ツ)\\_/¯
```

Reply:
Darn markdown... Yes, yes it should, feel free to make a PR for this

Reply:
Oh lol, if it's only that I can do it now